src/wp-includes/block-template-utils.php

```diff
@@ -725,17 +725,22 @@ function _wp_build_title_and_description_for_taxonomy_block_template( $taxonomy,
 }
 
 /**
- * Builds a unified template object based a post Object.
+ * Build a block template object from a post object.
  *
- * @since 5.9.0
- * @since 6.3.0 Added `modified` property to template objects.
- * @since 6.4.0 Added support for a revision post to be passed to this function.
+ * @since 6.5.0
  * @access private
  *
- * @param WP_Post $post Template post.
+ * @param WP_Post $post  Template post.
+ * @param array   $terms Additional terms to inform the template object.
+ * @param array   $meta  Additional meta fields to inform the template object.
  * @return WP_Block_Template|WP_Error Template or error object.
```

Comment:
This is now incorrect, it never seems to return an error object

Reply:
@swissspidy perhaps I'm misunderstanding. But on the opening line of the function we do some checks for required arguments and if not present we return an error.

Reply:
Argh, my bad, I looked further down & was on mobile. 🤦

Reply:
Phew! No problem haha! Thank you for double checking again 🙏🏻